test-lily/upload.py

```diff
@@ -32,6 +32,7 @@ def argv0_relocation ():
              'linux-ppc',
              'darwin-ppc',
              'darwin-x86',
+             'darwin-64',
```

Comment:
@Jahrme, as Darwin64 can't be built on GNU/Linux yet, what do you think about replacing this list with a dictionary with current list a keys and supported building host OS/architecture as values? I'd provide a patch, but I don't know which targets you can easily build on Darwin 64, is it everything listed here, or only Darwin 64, or something in between?